README.rst

```diff
@@ -529,6 +529,21 @@ Example: place the following in your ``~/.bashrc``
 
 	SOPS_GPG_EXEC = 'your_gpg_client_wrapper'
 
+
+Specify a different GPG key server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, ``sops`` uses the key server ``gpg.mozilla.org`` to retrieve GPG keys.
```

Comment:
I'd note that it does this only if the key is not found in the local keyring

Reply:
Thanks for the review @autrilla. Makes sense, let me add that.